client/src/theme.js

```diff
@@ -37,6 +39,10 @@ let theme = createTheme({
       ],
     },
   },
+  tab: {
+    minWidth: 50, // a number of your choice
+    width: 50, // a number of your choice
+}
 });
 
 theme = responsiveFontSizes(theme);
```

Comment:
probably want to get out of this file for this PR

Reply:
Taken care of in the next commit.